frontend/www/js/omegaup/components/common/Navbar.vue

```diff
@@ -68,82 +68,103 @@
           <li v-bind:class="{ active: navbarSection === 'contests' }" v-else="">
             <a href="/arena/" data-nav-arena>{{ T.wordsContests }}</a>
           </li>
+          <li
+            class="nav-courses"
+            v-bind:class="{ active: navbarSection === 'courses' }"
+          >
+            <a href="/schools/">{{ T.navCourses }}</a>
+          </li>
           <li
             class="dropdown nav-problems"
             v-bind:class="{ active: navbarSection === 'problems' }"
-            v-if="isLoggedIn && isMainUserIdentity"
           >
-            <a class="dropdown-toggle" data-toggle="dropdown" href="#"
-              ><span>{{ T.wordsProblems }}</span> <span class="caret"></span
-            ></a>
+            <a
+              class="dropdown-toggle"
+              data-toggle="dropdown"
+              data-nav-problems
+              href="#"
+            >
+              <span>{{ T.wordsProblems }}</span> <span class="caret"></span>
+            </a>
             <ul class="dropdown-menu">
               <li>
-                <a href="/problem/new/">{{ T.myproblemsListCreateProblem }}</a>
-              </li>
-              <li>
-                <a href="/problem/mine/">{{ T.navMyProblems }}</a>
-              </li>
-              <li>
-                <a href="/problem/">{{ T.wordsProblems }}</a>
+                <a href="/problem/" data-nav-all-problems>{{
```

Comment:
se puede mantener el prefijo de la sección? creo que esto se me fue en el review pasado.

```suggestion
                <a href="/problem/" data-nav-problems-all>{{
```

Reply:
acabo de mandar https://github.com/omegaup/omegaup/pull/3741 para arreglar esta inconsistencia en el otro menú.

Reply:
Listo!